components/content-blocks/Callout/CalloutEntry/index.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable react/prop-types */
+
+import PropTypes from "prop-types";
+import { useTranslation } from "react-i18next";
+import { makeDateString, useGlobalData } from "@/lib/utils";
+import { Text } from "../styles";
+import {
+  Section,
+  Inner,
+  SharePopup,
+  ImageWrapper,
+  ImageSticker,
+} from "./styles";
+import { Image } from "@rubin-epo/epo-react-lib";
+
+const getDateString = (newsDate, eventStart, eventEnd, lang) => {
```

Comment:
lil date making helper to negotiate news post date vs event date vs event date with start & end dates